java/src/jmri/jmrix/tams/TamsTrafficController.java

```diff
@@ -26,7 +26,8 @@
  *
  * Based on work by Bob Jacobsen and Kevin Dickerson
  *
- * @author Jan Boen
+ * @author  Jan Boen
+ * @version 20160922
```

Comment:
As @mattharris said, please remove the line--the entire change history of this file is available without this line. JMRI does not use the ```@version``` JavaDoc tags (we used to when it was possible to have CVS update this automatically, but not anymore).